src/integrations/BingAds/browser.js

```diff
@@ -1,6 +1,9 @@
 import logger from '../../utils/logUtil';
 import { LOAD_ORIGIN } from '../../utils/ScriptLoader';
 import { NAME } from './constants';
+import { buildCommonPayload, buildEcommPayload, EXCLUSION_KEYS } from './utils';
+import { removeUndefinedAndNullValues } from '../../utils/commonUtils';
+import { extractCustomFields } from '../../utils/utils';
```

Comment:
Importing only the relevant functions instead of all functions is good because it reduces unnecessary code imports and improves clarity